### DISCUSSION TITLE
marten migrate --dry-run (or --preview)

### DISCUSSION THREAD

**Discussion by carcinocron** [martenframework/marten#60] [Ideas]
Sometimes I'd like to see the outputted SQL before actually running it.

**Comment by ellmetha** [comment]
Hey! 👋 

Thanks for the idea!

I have to see if this is something that would be possible: technically SQL statements could be initiated by `run_code` or `run_sql` operations as well, so this makes this a bit tricky to implement.

An alternative option (and probably a more readable one) would be to simply output the description of each operation contained in the migrations that need to be applied. Do you think that would be something helpful?
